profiles/factory.py

```diff
@@ -4,7 +4,7 @@
 from profiles.api.errors import OAuth2Error, ClientError
 from profiles.api.oauth2 import OAUTH2_BP
 from profiles.api.ping import PING_BP
-from profiles.utilities import remove_none_values, chain_exception
+from profiles.utilities import chain_exception, remove_none_values
```

Comment:
Any way to enforce this in Pyline @giorgiosironi?

Reply:
This may be considered style rather than static analysis, as the semantics of the two are the same; should check out https://github.com/elifesciences/builder/blob/master/.scrub.sh#L11